Can't build when moduleResolution set to bundler.

my `tsconfig.json`:
```json
{
    "compilerOptions": {
        "allowJs": true,
        "baseUrl": "./",
        "module": "ESNext",
        "moduleResolution": "bundler",
        "jsx": "react-jsx",
        "strict": true,
        "isolatedModules": true,
        "target": "ESNext",
        "esModuleInterop": true,
        "forceConsistentCasingInFileNames": true,
        "noEmit": true,
        "skipLibCheck": true,
        "paths": {
            "@/*": [
                "./resources/js/*"
            ],
            "@styles/*": [
                "./resources/css/*"
            ]
        }
    },
    "include": [
        "resources/js/**/*.ts",
        "resources/js/**/*.tsx",
        "resources/js/**/*.d.ts"
    ],
    "exclude": [
        "node_modules/*"
    ]
}
```

tsc error:
```
yarn run v1.22.19
$ tsc && vite build && vite build --ssr
resources/js/Pages/Articles/Show/index.tsx:5:25 - error TS7016: Could not find a declaration file for module 'rehype-prism-plus'. '/home/dairidong/code/react_blog/node_modules/rehype-prism-plus/dist/rehype-prism-plus.es.js' implicitly has an 'any' type.
  There are types at '/home/dairidong/code/react_blog/node_modules/rehype-prism-plus/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'rehype-prism-plus' library may need to update its package.json or typings.

5 import rehypePrism from "rehype-prism-plus";
```
it's only can run success if i set `"moduleResolution": "node"` or  `"noImplicitAny": false`

--- a/package.json
+++ b/package.json
@@ -11,10 +11,22 @@
   "types": "./dist/index.d.ts",
   "type": "module",
   "exports": {
-    ".": "./dist/rehype-prism-plus.es.js",
-    "./common": "./dist/common.es.js",
-    "./all": "./dist/all.js",
-    "./generator": "./dist/generator.es.js"
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/rehype-prism-plus.es.js"
+    },
+    "./common": {
+      "types": "./dist/src/common.d.ts",
+      "import": "./dist/common.es.js"
+    },
+    "./all": {
+      "types": "./dist/src/all.d.ts",
+      "import": "./dist/all.es.js"
+    },
+    "./generator": {
+      "types": "./dist/src/generator.d.ts",
+      "import": "./dist/generator.es.js"
+    }
   },
   "typesVersions": {
     "*": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "rehype-prism-plus",
-  "version": "1.6.1",
+  "version": "1.6.2",
   "description": "rehype plugin to highlight code blocks in HTML with Prism (via refractor) with line highlighting and line numbers",
   "source": "index.js",
   "files": [

